### ISSUE TITLE
Loss function missing in first notebook

### ISSUE THREAD

**Issue by vinsis** [acids-ircam/pytorch_flows#1]
In `flows_01.ipynb`, the cell below the text 
`Now the only missing ingredient is the loss function that is simply defined as follows`
is empty. I believe the loss function was defined here. 

Can you please share the loss function? Thanks.

**Comment by vinsis** [comment]
Found it in the previous commit:

```python
def loss(density, zk, log_jacobians):
    sum_of_log_jacobians = sum(log_jacobians)
    return (-sum_of_log_jacobians - torch.log(density(zk) + 1e-9)).mean()
```
